examples/chart/teleport-cluster/charts/teleport-operator/operator-crds/resources.teleport.dev_accesslists.yaml

```diff
@@ -194,16 +194,8 @@ spec:
                 description: Conditions represent the latest available observations
                   of an object's state
                 items:
-                  description: "Condition contains details for one aspect of the current
-                    state of this API Resource.\n---\nThis struct is intended for
-                    direct use as an array at the field path .status.conditions.  For
-                    example,\n\n\n\ttype FooStatus struct{\n\t    // Represents the
-                    observations of a foo's current state.\n\t    // Known .status.conditions.type
-                    are: \"Available\", \"Progressing\", and \"Degraded\"\n\t    //
-                    +patchMergeKey=type\n\t    // +patchStrategy=merge\n\t    // +listType=map\n\t
-                    \   // +listMapKey=type\n\t    Conditions []metav1.Condition `json:\"conditions,omitempty\"
-                    patchStrategy:\"merge\" patchMergeKey:\"type\" protobuf:\"bytes,1,rep,name=conditions\"`\n\n\n\t
-                    \   // other fields\n\t}"
+                  description: Condition contains details for one aspect of the current
```

Comment:
What causes all of these yaml changes?

Reply:
The CRDs are generated from two sources:
- CR spec comes from our proto definitions
- CR status comes from go structs importing kubernetes structs like the conditions. The descriptions are coming from the godoc of those upstream structs

When we update the kube version, we also update those structs. If they changed the godocs, it triggers those changes.

tl;dr it's safe an expected, I went through the diffs and it's only on status fields coming from upstream

Reply:
So this is a consequence of the k8s version bump? Got it.

Reply:
Yes. `sigs.k8s.io/controller-tools` is responsible for generating the CRD descriptions and auto-added these fixed descriptions to the `status` field of the objects.

In the most recent version, the change removed the extra and unrelated description. Seems much better now that it doesn't reference FooStatus 🙃